### PR TITLE
Fixed minor bugs in pydy.org 

### LINKED ISSUE #17: The responisive design needs some work.
The logo doesn't work with the responsive design and the side bars are placed above the intro:
pydy multibody dynamics with python pydy website - mozilla firefox_060

### LINKED ISSUE #18: Logo too small
The logo is kinda small; can we make it bigger?

### LINKED ISSUE #19: The logo does not appear on the examples page
http://www.pydy.org/examples/examples.html

### PR DESCRIPTION
**Pull request by sahilshekhawat**
Fixes #17 Fixes #18 Fixes #19

### CODE REVIEW

File: source/_theme/pydy/layout.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {% extends "basic/layout.html" %}
 
-{% set theme_css_files = ['_static/main.css', '_static/custom.css',
+{% set theme_css_files = ['/_static/main.css', '/_static/custom.css',
```
Comment (moorepants) [inline source/_theme/pydy/layout.html]: I don't agree with this change. Can you prove to me why it is correct?

I've built numerious Sphinx sites and plenty of of static sites without having to do this.

I don't think you should be required to view locally with a server.

Comment (sahilshekhawat) [inline source/_theme/pydy/layout.html]: Yes, only logo needs to be changed. I didn't knew that sphinx automatically handles headers. This is incorrect.

File: source/_theme/pydy/layout.html [hunk 2]
```diff
@@ -36,17 +36,25 @@
   <div class="main-nav" id="main-nav">
     <div id="site-desc" class="site-desc">
       <div class="logo">
-        <img class="image"src="_static/logo.svg">
+        <img class="image"src="/_static/logo.svg">
```
Comment (moorepants) [inline source/_theme/pydy/layout.html]: This still will not work locally without a server.

Comment (sahilshekhawat) [inline source/_theme/pydy/layout.html]: Yes but Currently sphinx is not handling it. Thus, logo is not displayed in some pages.

Comment (moorepants) [inline source/_theme/pydy/layout.html]: Should we be using this: http://sphinx-doc.org/templating.html#logo in a Jinja template?

**Pull request merged**